pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -36,12 +37,12 @@ func Add(mgr manager.Manager) error {
 }
 
 // newReconciler returns a new reconcile.Reconciler
-func newReconciler(mgr manager.Manager) reconcile.Reconciler {
+func newReconciler(mgr manager.Manager) *ReconcileCheInstallation {
```

Comment:
why do you want to change the returned type here?

Reply:
To make this statement compile: https://github.com/nurali-techie/toolchain-operator/pull/1/files?file-filters%5B%5D=.go#diff-b09f4db8198fe6aeba0154bd7f8fdfbeR76

Reply:
ok, got it.